docs/benchmarks/overview.md

```diff
@@ -1,3 +1,30 @@
-# Overview
+## Set Up Benchmarking
 
-This is an overview of the benchmarking section.
\ No newline at end of file
+Before initiating the benchmarking process, it's essential to consider the method of GPU configuration. There are two primary approaches available: running an Ansible playbook designed for GPU setup and manually executing bash commands. Each approach presents distinct advantages and considerations that should be taken into account to make an informed choice for benchmarking.
+
+
+### Running Ansible Playbook for GPU Configuration:
+
+#### Advantages:
+Automation: Ansible playbooks allow for automated and repeatable GPU configuration, ensuring consistency and reducing the risk of human errors.
+Scalability: Ansible is designed to manage configurations across multiple machines, making it efficient for large-scale deployments.
+Collaboration: Playbooks can be shared and reused, enabling collaborative configurations.
+
+#### Considerations:
+Learning Curve: Familiarity with Ansible's syntax and structure might be necessary.
+Initial Setup: Setting up Ansible and creating playbooks might require upfront investment in time.
+
+
+### Manual Execution of Bash Commands for GPU Configuration:
```

Comment:
@lovellainso For benchmarking I thought we are going to run notebooks on Jupyter server instead of bash commands?

Reply:
Nevermind my bad, I overlooked and thought this is about the actual testing. Resolving this.